index.js

```diff
@@ -191,12 +213,16 @@ FocusGroup.prototype._findIndexOfNode = function(searchNode) {
 };
 
 FocusGroup.prototype._getActiveElementIndex = function() {
-  return this._findIndexOfNode(document.activeElement);
+  return this._rootNode !== document
+    ? this._activeIndex
+    : this._findIndexOfNode(document.activeElement);
 };
 
 FocusGroup.prototype.focusNodeAtIndex = function(index) {
   var member = this._members[index];
   if (member) focusNode(member.node);
```

Comment:
Before I get back into this, I wanted to make sure I understand what you mean here. If I'm understanding correct, you'd want the focus group to contain focusable items, as well as regular items?

I've been thinking about it and I'm not sure if that would work. For the fake focus to work an item needs to hold focus like an input or button. Native focusable items can be included in the fake focus group now, it just won't receive a true focus, but the fake one instead.